web_generator/lib/src/bcd.dart

```diff
@@ -84,7 +90,7 @@ class BrowserCompatData {
   BCDInterfaceStatus? retrieveInterfaceFor(String name) => interfaces[name];
 
   bool shouldGenerateInterface(String name) =>
-      retrieveInterfaceFor(name)?.shouldGenerate ?? false;
+      generateAll || (retrieveInterfaceFor(name)?.shouldGenerate ?? false);
```

Comment:
if we can't remove the uses of `generateAll` from BCDInterfaceStatus/BCDPropertyStatus, maybe we could remove this one and the one in `isEventHandlerSupported` (seems that we'll get `true` from the underlying values, except when there isn't a interface for name (e.g. null result?)